### PR TITLE
[params] Fixes provider admin controllers

### PR DESCRIPTION
**Pull request by nidhi-soni1104**
**What this PR does / why we need it:**
Fixes usage of params in the provider admin controllers

### CODE REVIEW

File: app/controllers/provider/admin/account/change_plans_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def show
   end
 
   def update
-    plan = plans.find(params[:change_plan][:plan_id])
+    plan = plans.find(params.permit(:change_plan)[:plan_id])
```
Comment (hallelujah) [inline app/controllers/provider/admin/account/change_plans_controller.rb:8]: You have to permit the subkeys too

See https://api.rubyonrails.org/v5.0.2/classes/ActionController/Parameters.html

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: app/controllers/provider/admin/account/change_plans_controller.rb (hunk 1 above)
Comment (nidhi-soni1104) [reply app/controllers/provider/admin/account/change_plans_controller.rb]: @hallelujah permitted the used ones

**Review by hallelujah — CHANGES_REQUESTED** [review]
Please read and understand well  https://api.rubyonrails.org/v5.0/classes/ActionController/Parameters.html 

Hash needs to permit all given attributes

File: app/controllers/provider/admin/webhooks_controller.rb [hunk 2]
```diff
@@ -42,7 +42,7 @@ def create
   end
 
   def update
-    if @webhook.update_attributes(params[:web_hook])
+    if @webhook.update_attributes(params.permit(:web_hook)[:web_hook])
```
Comment (hallelujah) [inline app/controllers/provider/admin/webhooks_controller.rb:45]: Same as in https://github.com/3scale/porta/pull/2833/files#r771256105

File: app/controllers/provider/admin/webhooks_controller.rb [hunk 3]
```diff
@@ -31,7 +31,7 @@ def show
   end
 
   def create
-    @webhook ||= current_account.build_web_hook(params[:web_hook])
+    @webhook ||= current_account.build_web_hook(params.permit(:web_hook)[:web_hook])
```
[the remark this answers is not in the document]
Comment (nidhi-soni1104) [reply app/controllers/provider/admin/webhooks_controller.rb:34]: The confusion in permitting keys is I permit which I see being used in controller, I think need to check on view to know the list of keys.

**Pull request closed without merging**